### PR TITLE
fix(efb): Fix Scrolling in EFB Loadsheet

### PR DESCRIPTION
**Pull request by akhileshmalini**
Fixes Issue where scrolling load sheet is impossible.

## Summary of Changes

Adds a scrollbar to the side that allows users can drag. 

## Screenshots (if necessary)

Screen Shot 2021-03-02 at 8 07 33 AM
Screen Shot 2021-03-02 at 8 06 20 AM

## Additional context

Discord username (if different from GitHub): EwilCorp

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: src/instruments/src/EFB/Dispatch/Pages/Loadsheet.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ .loadsheet-container{
+   height: 700px;
+}
+
+.show-scrollbar::-webkit-scrollbar {
+    width: 1em;
```
Comment (myselfaaryan) [inline src/instruments/src/EFB/Dispatch/Pages/Loadsheet.scss:24]: @apply w-4;

File: src/instruments/src/EFB/Dispatch/Pages/LoadsheetPage.tsx [hunk 2]
```diff
@@ -26,7 +28,10 @@ const LoadSheetWidget = (props: LoadsheetPageProps) => {
     return (
         <div className="px-6">
             <div className="w-full">
-                <div className="bg-gray-800 rounded-xl p-6 text-white shadow-lg mr-4 overflow-y-scroll" dangerouslySetInnerHTML={{__html: props.loadsheet}}>
+
+				<div className="bg-gray-800 rounded-xl p-6 text-white shadow-lg mr-4">
```
Comment (myselfaaryan) [inline src/instruments/src/EFB/Dispatch/Pages/LoadsheetPage.tsx:32]: indents

File: src/instruments/src/EFB/Dispatch/Pages/Loadsheet.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+.show-scrollbar::-webkit-scrollbar-track {
+	box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
+    -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,0.3);
```
Comment (myselfaaryan) [inline src/instruments/src/EFB/Dispatch/Pages/Loadsheet.scss:29]: indents

**Review by myselfaaryan — CHANGES_REQUESTED** [review]

**Comment by Benjozork** [comment]
Styling needs improvement imo. Maybe add some FBW blue (called blue-light in the tailwind config) and some ever so slightly rounded corners ?

**Comment by akhileshmalini** [comment]
Updated Scrollbar styles

**Review by ZigTag — APPROVED** [review]
LGTM

**Comment by akhileshmalini** [comment]
Based on conversations on Discord, I've added a way for people to increase Font size.

**Comment by marcman86** [comment]
Name: marcman86#4907
Date of testing: 17.2.2021
Version of the sim: 1.13.16.0
PR Tested: #3519
Tier of testing: 1
Changes to observe: EFB Loadsheet scrolling
Testing technique: Tryout

Steps made to test the Pull Request:

1.) Spawned C&D at EDDP
2.) Started EFB
3.) Tested loadsheet scrolling via bar and touchscreen drag
4.) Tested min. zoom (Image1) and max. zoom (Image 2)

Media:


Image1


Image2

Issues: 

- The slider is tiny and the sliding bar very thin, so the clickzones are tiny to. This gets frustrating cause the EFB is small for itself.
- The horizontal space is not fully used on max zoom. 
- Horizontal space is wasted from the two zoom buttons. 
- Images dont get magnified when zoomed in.

Design suggestions (my opinion :-)): 

- Make the slider size as a zoom button and also the slidebar in this width. 
- Shorten the slidebar and put the zoom buttons under the slidebar.
- Make max. zoom higher, so that the text will fill the whole page.

Minor improvement: Can you make the (mouse)hand grab on drag? 

Overall Rating: -

Conclusions of the testing: Needs to be improved.

Hope this helps :-)

**Comment by FlyingCarpet7** [comment]
When you are at this page can you please change the Loadsheet menu header to Flightplan or OFP.

**Comment by akhileshmalini** [comment]
I've fixed the issue with the images and I've allowed for more zoom. 


As for the size of the  scrollbar, I'm not entirely sure how we would achieve that. But the Touch and scroll should be helpful in certain scenarios. 

Not sure if we can move the zoom buttons under the scrollbar. If anything, that'll make it even harder to scroll because of the reduced height of the scrollbar.

**Comment by akhileshmalini** [comment]
And changing the hand to grab, I've actually implemented that in css, but it doesn't seem to work.

**Comment by Benjozork** [comment]
Please fix your lint errors

**Pull request closed without merging**